waku/v2/node/rest/client.nim

```diff
@@ -0,0 +1,4 @@
+import presto/client
```

Comment:
Top level `{.push raises: [Defect].}` for all modules. Reasoning [here](https://status-im.github.io/nim-style-guide/errors.exceptions.html).